tests/common/response.py

```diff
@@ -1,4 +1,6 @@
 def assert_dict_struct(data, structure):
+    print("data", data)
```

Comment:
@bradleySuira please remove this prints :)

Reply:
fixed